### ISSUE TITLE
1.20.1 Forge 47.2.7

### ISSUE THREAD

**Issue by Krimatoria** [Tau-ri-Dev/Mod-JSG-1.20.x#36]
jsg-1.20.1-5.0.1.0-Alpha.jar

> > [21:06:27] [Worker-ResourceReload-4/WARN] [minecraft/ModelBakery]: Unable to load model: 'jsg:invisible_block#inventory' referenced from: jsg:invisible_block#inventory: java.io.FileNotFoundException: jsg:models/item/invisible_block.json
> > [21:06:27] [Worker-ResourceReload-4/WARN] [minecraft/ModelBakery]: Unable to load model: 'jsg:iris_block#inventory' referenced from: jsg:iris_block#inventory: java.io.FileNotFoundException: jsg:models/item/iris_block.json
> [21:06:12] [Worker-ResourceReload-5/WARN] [minecraft/SoundManager]: File jsg:sounds/misc/dialer/button_error_universe.ogg does not exist, cannot add it to event jsg:universe_dialer_error
> [21:06:12] [modloading-worker-0/WARN] [fu.ni.co.ConfigSpecWrapper/CORE]: Configuration file C:\Users\Dotka\AppData\Roaming\PrismLauncher\instances\1.20.1(1)\.minecraft\config\jsg\jsgConfig_1.0\Stargate.toml is not correct. Correcting

**Comment by MineDragonCZ** [comment]
Whats wrong here?

**Comment by Krimatoria** [comment]
> Что здесь не так?

Are you asking me? This is your mod)))

**Comment by TheRedCreations** [comment]
Please upload the full log file here. If the game crashed, there should be a crash report. Upload that too please.

**Comment by Krimatoria** [comment]
> Please upload the full log file here. If the game crashed, there should be a crash report. Upload that too please.

There is no crash report) I'm just checking for other errors related to other mods in the build by looking at the log file while Minecraft is running.

I was looking for a solution to this) And I've already found problematic mods. 
Sound Engine and [software.bernie.gecko like.GeckoLib/]: Unable to parse animation: attack

But I saw warnings with your mod in the log and decided to send it to you. I don't think it's a critical issue yet, it's just a warning. I can try to upload the log later.
